### PR TITLE
Logged how many losses are being saved in the event_loss_asset table

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1429819.
The tests are running here: https://ci.openquake.org/job/zdevel_oq-engine/1086/

### CODE REVIEW

File: openquake/engine/calculators/risk/event_based_risk/core.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def event_based(workflow, getter, outputdict, params, monitor):
     # NB: event_loss_table is a dictionary (loss_type, out_id) -> loss,
     # out_id can be None, and it that case it stores the statistics
     event_loss_table = {}
+    num_losses = collections.defaultdict(lambda : numpy.zeros(2, dtype=int))
```
Comment (ptormene) [inline openquake/engine/calculators/risk/event_based_risk/core.py]: Please add a comment

File: openquake/engine/calculators/risk/event_based_risk/core.py [hunk 2]
```diff
@@ -105,6 +106,8 @@ def event_based(workflow, getter, outputdict, params, monitor):
                                     event_loss=event_loss, rupture_id=rup_id,
                                     asset=asset, loss=loss_per_rup)
                                 inserter.add(ela)
+                            num_losses[asset.asset_ref] += numpy.array(
```
Comment (ptormene) [inline openquake/engine/calculators/risk/event_based_risk/core.py]: Please add a comment

**Comment by ptormene** [comment]
If you add some comments, LGTM

File: openquake/engine/calculators/risk/event_based_risk/core.py (hunk 1 above)
Comment (micheles) [inline openquake/engine/calculators/risk/event_based_risk/core.py]: Added

File: openquake/engine/calculators/risk/event_based_risk/core.py (hunk 2 above)
Comment (micheles) [inline openquake/engine/calculators/risk/event_based_risk/core.py]: Added

**Pull request merged**